client/app/project/project.html

```diff
@@ -126,18 +125,6 @@
                             <div ng-show="projectCtrl.mappingStep === 'viewing' ">
                                 <!-- task information -->
                                 <h2>Mapping</h2>
-                                <h3>Task
-                                    #{{ projectCtrl.selectedTaskData.taskId }}
-                                    is {{ projectCtrl.selectedTaskData.taskStatus }}
-                                    <span ng-show="projectCtrl.selectedTaskData.taskStatus === 'LOCKED_FOR_MAPPING' || projectCtrl.selectedTaskData.taskStatus === 'LOCKED_FOR_VALIDATION' "> by
-                                         <span ng-show="projectCtrl.selectedTaskData.lockHolder === projectCtrl.user.username">
-                                            you
-                                         </span>
-                                         <span ng-show="projectCtrl.selectedTaskData.lockHolder !== projectCtrl.user.username">
-                                            {{ projectCtrl.selectedTaskData.lockHolder }}
-                                         </span>
-                                    </span>
-                                </h3>
```

Comment:
We should put the lockholder as an item in advanced table, so that you can see who has something locked